app/experimenter/nimbus-ui/src/components/TableVisualizationRow/significance-neutral.svg

```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M20 12c0 .6-.4 1-1 1H5c-.6 0-1-.4-1-1s.4-1 1-1h14c.6 0 1 .4 1 1z" fill="#212529"/><mask id="a" maskUnits="userSpaceOnUse" x="4" y="11" width="16" height="2"><path d="M20 12c0 .6-.4 1-1 1H5c-.6 0-1-.4-1-1s.4-1 1-1h14c.6 0 1 .4 1 1z" fill="#fff"/></mask></svg>
```

Comment:
Ooo good question, I know `rapid` was using font awesome which we haven't used in `nimbus`. I used SVGs since they were a copy-and-paste from the icons shown in Figma, to be fair I didn't think to look at Bootstrap icons, but it looks like they're a different package/dependency...

https://getbootstrap.com/docs/4.5/extend/icons/

> While you won’t find a built-in icon library in Bootstrap, our separate Bootstrap Icons projects is a growing set of open source SVGs you can use. While they’re designed first and foremost to work with our components and documentation, you can use them in any project.

I suppose we could install the package, but it gives us another dependency, we won't use most of them, and I'm guessing there's icons in the navigation that aren't in this library (I could be wrong). The other option is to download them separately as SVGs in which case we might as well use what's in Figma? 🤔

Reply:
Oh cool. Using svg's makes sense in this case it seems then. Thanks for looking into it!